package.json

```diff
@@ -47,6 +47,7 @@
     "redux-saga": "^1.1.3",
```

Comment:
<img width="289" alt="Screenshot 2021-06-09 at 23 18 21" src="https://user-images.githubusercontent.com/2698505/121423475-fd6d7680-c978-11eb-81c5-4a4b146ab7ec.png">

The branch format is `FAN-000_short-description`. It's not required to change it right now, but plz pay attention to it since we need to understand how pull requests and jira issues are releated